soul-admin/src/main/java/org/dromara/soul/admin/dto/SelectorDTO.java

```diff
@@ -81,4 +85,21 @@
      * selector conditions.
      */
     private List<SelectorConditionDTO> selectorConditions;
+
+    @Builder
+    private SelectorDTO(final String id, final String pluginId, final String name, final Integer matchMode, final Integer type,
+                       final Integer sort, final Boolean enabled, final Boolean loged, final Boolean continued, final String handle,
+                       final List<SelectorConditionDTO> selectorConditions) {
+        this.id = id;
+        this.pluginId = pluginId;
+        this.name = name;
+        this.matchMode = matchMode;
+        this.type = type;
+        this.sort = sort;
+        this.enabled = enabled;
+        this.loged = loged;
+        this.continued = continued;
+        this.handle = handle;
+        this.selectorConditions = selectorConditions;
+    }
```

Comment:
why not use the annotation AllArgsConstructor?

Reply:
Yeah, this one doesn't have a parent class and I don't have to do that, so Let me change it